### ISSUE TITLE
Share (sub)list

### ISSUE THREAD

**Issue by orbifx** [orgzly/orgzly-android#69]
It would be nice having the ability to quickly select a notebook, list or sublist and share it as plain text with another application.

**Comment by DamienCassou** [comment]
I agree. For example, I would like to send a list by SMS to my spouse.

**Comment by fxshpac** [comment]
Just add my vote on this. I just need to share the body text of a note to other apps. For now, I can select all text, and press Share after but it will be much faster doing it in one step.
